doc/manual.adoc

```diff
@@ -203,6 +203,8 @@ following variables.
 |%(repo:is-inside-work-tree)
 			|Whether Tig is running inside a work tree,
 			 either `true` or `false`.
+|%(repo:hash_len)	|The hash algorithm used for the repository, e.g. `sha1`
+			 or `sha256`.
```

Comment:
Maybe this should be called `%(repo:extensions.objectformat)` to use the standard Git term.
Actually, is there even a need to expose this at all?
I don't think we expose any other config options. Users can already consult something like
`git --git-dir=%(repo:git-dir) config extensions.objectformat`

Reply:
Thanks for the review @krobelus. %(repo:hash_len)	is actually a by-product of storing hash_len in the repo struct, but it's the best compromise I have found. I'll make it more elegant.

Reply:
oh it is exposed by the `REPO_INFO` macro. Maybe define it outside that